pkg/template/processor_test.go

```diff
@@ -0,0 +1,604 @@
+		require.NoError(t, err)
+		err = p.Apply(objs, 2*time.Second)
+
+		// then
+		require.NoError(t, err)
+		verifyNamespace(t, cl, namespace)
+	})
+
+	t.Run("should create role binding alone", func(t *testing.T) {
+		// given
+		s := addToScheme(t)
+		namespace, commit, user := templateVars()
+		values := paramsKeyValues(namespace, commit, user)
+
+		cl := test.NewFakeClient(t)
+		p := template.NewProcessor(cl, s)
+
+		// when
+		objs, err := p.Process(templateContent(roleBindingObj), values)
+		require.NoError(t, err)
+		err = p.Apply(objs, 2*time.Second)
+
+		// then
+		require.NoError(t, err)
+		verifyRoleBinding(t, cl, namespace)
+	})
+
+	t.Run("should create namespace and role binding", func(t *testing.T) {
+
+		t.Run("success", func(t *testing.T) {
+			// given
+			s := addToScheme(t)
+			namespace, commit, user := templateVars()
+			values := paramsKeyValues(namespace, commit, user)
+			cl := test.NewFakeClient(t)
+			p := template.NewProcessor(cl, s)
+
+			// when
+			objs, err := p.Process(templateContent(namespaceObj, roleBindingObj), values)
+			require.NoError(t, err)
+			err = p.Apply(objs, 2*time.Second)
+
+			// then
+			require.NoError(t, err)
+			verifyNamespace(t, cl, namespace)
+			verifyRoleBinding(t, cl, namespace)
+		})
+
+		t.Run("failure", func(t *testing.T) {
+
+			t.Run("client error", func(t *testing.T) {
+
+				t.Run("namespace does not exist", func(t *testing.T) {
+					// given
+					s := addToScheme(t)
+					namespace, commit, user := templateVars()
+					values := paramsKeyValues(namespace, commit, user)
+
+					cl := test.NewFakeClient(t)
+					cl.MockCreate = func(ctx context.Context, obj runtime.Object) error {
+						if obj.GetObjectKind().GroupVersionKind().Kind == "RoleBinding" {
+							return errors.New("mock error")
+						}
+						return nil
+					}
+					p := template.NewProcessor(cl, s)
+
+					// when
+					objs, err := p.Process(templateContent(namespaceObj, roleBindingObj), values)
+					require.NoError(t, err)
+					err = p.Apply(objs, 2*time.Second)
+
+					// then
+					require.Error(t, err)
+				})
+			})
+
+		})
+	})
+
+	t.Run("should update existing role binding", func(t *testing.T) {
+		// given
+		s := addToScheme(t)
+		namespace, commit, user := templateVars()
+		values := paramsKeyValues(namespace, commit, user)
+
+		cl := test.NewFakeClient(t)
+		p := template.NewProcessor(cl, s)
+
+		objs, err := p.Process(templateContent(roleBindingObj), values)
+		require.NoError(t, err)
+		err = p.Apply(objs, 2*time.Second)
+
+		require.NoError(t, err)
+		verifyRoleBinding(t, cl, namespace)
+
+		// when
+		objs, err = p.Process(templateContent(roleBindingObj, newUser), values)
+		require.NoError(t, err)
+		err = p.Apply(objs, 2*time.Second)
+
+		// then
+		require.NoError(t, err)
+		binding := verifyRoleBinding(t, cl, namespace)
+		require.Len(t, binding.Subjects, 2)
+	})
+
+	t.Run("should fail to create template object", func(t *testing.T) {
```

Comment:
this test looks very similar to this one: https://github.com/codeready-toolchain/member-operator/pull/49/files#diff-a5ef365842f1e1803e47926eaf743087R435

Reply:
I'm sorry, the link brings me to other review comments. Could you please give me the title of the test that looks similar to `should fail to create template object` ?